src/server.js

```diff
@@ -215,5 +224,91 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_ADDRESS}/geocode`,
+  async (request, response) => {
+    let address = request.params[URL_PARAM_ADDRESS];
+
+    let geocodeRequest =
+      "https://maps.googleapis.com/maps/api/geocode/json?address=" +
+      address +
+      "&key=" +
+      env.API_KEY;
+
+    try {
+      const geocodeResponse = await (await fetch(geocodeRequest)).json();
+      const geocodeResponseStatus = geocodeResponse.status;
+
+      if (geocodeResponseStatus !== "OK") {
+        console.error(
+          "Geocoding error occured. Api response status: " +
+            geocodeResponseStatus
+        );
+        response
+          .status(500)
+          .json({ status: 500, error: { type: geocodeResponseStatus } });
+      } else {
+        response.json({
+          status: 200,
+          data: geocodeResponse.results[0].geometry.location,
```

Comment:
Why results[0], does the geocode API respond with multiple locations?

Reply:
as a follow up, is it possible for the geocode response to contain 0 results for any reason?

Reply:
Yes, the Geocode API can respond with multiple locations. This happens when the submitted address is not specific enough and multiple locations are found. It is also possible to get 0 results if there aren't any locations matching the input from the user. 

Currently, if there are no results, geocodeResponseStatus is "ZERO_RESULTS" rather than "OK". The status is returned to the client as an error. If for some reason the geocodeResponseStatus is "OK", but there are no results, an error would get thrown and caught, and returned to the client as a server error as well.